app/src/main/java/com/miguelgaeta/android_media_picker/AppActivity.java

```diff
@@ -111,4 +130,23 @@ public void onCancelled() {
     public Context getContext() {
         return this;
     }
+
+    private class MyContentFileProvider implements MediaPicker.ContentFileProvider {
```

Comment:
Should this be part of the library module as the default provider implementation?